packages/balancer/services/autonomous_fund_mainnet/service.yaml

```diff
@@ -0,0 +1,35 @@
+name: autonomous_fund_mainnet
+author: balancer
+version: 0.1.0
+description: A set of agents implementing the Autonomous Fund.
+aea_version: '>=1.0.0, <2.0.0'
+license: Apache-2.0
+fingerprint:
+  README.md: bafybeibfahh3htjez7vf7lwx2s7tth26cwxgtalgn5hj7yg7akhk67f4ny
+fingerprint_ignore_patterns: []
+agent: balancer/autonomous_fund:0.1.0:bafybeievg5lnqh6u4akac34cbrx52wywlue4x5kegq5d5j2ttzv3d7hbua
+number_of_agents: 1
+---
+public_id: balancer/autonomous_fund_abci:0.1.0
+type: skill
+models:
+  params:
+    args:
+      setup:
+        safe_contract_address: ${SAFE_CONTRACT_ADDRESS:list:["0xD1A2679e3455a018d9e0D11edb873e7C5B5d831e"]}
+      pool_tokens: ${POOL_TOKENS:list:["0x2f3A40A3db8a7e3D09B0adfEfbCe4f6F81927557","0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6","0xC04B0d3107736C32e19F1c62b2aF67BE61d63a05"]}
+      managed_pool_address: ${MANAGED_POOL_ADDRESS:str:0xEe1a504F8BAdc2c0215119cb729B8B6a0ecB13a1}
+      allowed_lp_addresses: ${ALLOWED_LP_ADDRESSES:list:[]}
+      enforce_allowlist: ${ENFORCE_ALLOWLIST:bool:false}
+      observation_interval: ${OBSERVATION_INTERVAL:int:600}
```

Comment:
Overriden the observation interval, its much shorter now (10min vs 20hrs), this should help for stability.